mlir/include/mlir/Dialect/AMDGPU/Utils/Chipset.h

```diff
@@ -49,6 +49,14 @@ struct Chipset {
 #undef DEFINE_COMP_OPERATOR
 };
 
+inline bool isGfx940Series(const Chipset &chipset) {
```

Comment:
Per recent codebase cleanup, `isGfx942Series` if we don't want to straight-up `==` it.

Reply:
Looks like this is supposed to match with all the gfx94_ chipsets, hence the "Series" in the function name. But if gfx940 and gfx941 are no longer supported, I can rename this to `isGfx942` and just check `chipset == Chipset(9, 4, 2)`.

Reply:
For that, there's just a `kGfx942` running around in a bunch of files - take equality to that

Reply:
And I'm not sure if "where needed" wasn't supposed to mean "static method in the `.cpp`".

Reply:
That makes sense.

@kuhar What do you think?

Reply:
In this instance we are pretty sure gfx942 is the only gfxip version in the gfx940 series